src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
+      <a href="#" class="stars__star"></a>
```

Comment:
This is not a link, so you need to fix it everywhere
```suggestion
      <div href="#" class="stars__star"></div>
      <div href="#" class="stars__star"></div>
      <div href="#" class="stars__star"></div>
      <div href="#" class="stars__star"></div>
      <div href="#" class="stars__star"></div>
```